src/api/stripe/stripe-checkout-redirect.ts

```diff
@@ -0,0 +1,36 @@
+import {loadStripe} from '@stripe/stripe-js'
+import {pickBy} from 'lodash'
+import axios from 'utils/configured-axios'
+
+if (!process.env.NEXT_PUBLIC_STRIPE_PUBLIC_KEY) {
+  throw new Error('no Stripe public key found')
+}
+
+const stripePromise = loadStripe(process.env.NEXT_PUBLIC_STRIPE_PUBLIC_KEY)
+
+const stripeCheckoutRedirect = async (priceId: string, email: string) => {
+  return await axios
+    .post(
+      `${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/api/v1/stripe/subscription`,
+      pickBy({
+        price_id: priceId,
+        site: 'egghead.io',
+        email: email,
+        client_id: process.env.NEXT_PUBLIC_CLIENT_ID,
+        success_url: `${process.env.NEXT_PUBLIC_AUTH_DOMAIN}?session_id={CHECKOUT_SESSION_ID}`,
```

Comment:
should this be:

```suggestion
        success_url: `${process.env.NEXT_PUBLIC_REDIRECT_URI}?session_id={CHECKOUT_SESSION_ID}`,
```

So the user comes back to next.egghead.io?